shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -138,6 +139,26 @@ export default {
     this.$store.dispatch('cluster/forgetType', MANAGEMENT.NODE);
   },
 
+  watch: {
+    // logic to be removed once kube version is 1.25 or greater
```

Comment:
I don't think we should plan to remove the logic because there could be a situation in the future where some Rancher managed clusters are running 1.25 and some are running an older version. So I think the if-statement should also check if the minor version is less than 25.

And the banner should only show if the major version is exactly equal to "1".

Reply:
updated conditions to:
`neu.status?.version?.major === 1 && neu.status?.version?.minor >= 21 && neu.status?.version?.minor < 25`

Reply:
Updated conditions based on your feedback 🙏 

`if (major === 1 && minor >= 21 && minor < 25)`